client/my-sites/plugins/main.jsx

```diff
@@ -386,6 +387,7 @@ const PluginsMain = React.createClass( {
 		return (
 			<Main className={ containerClass }>
 				{ this.renderDocumentHead() }
+				<QuerySites allSites />
```

Comment:
Gotcha @johnHackworth I suppose if it's in there already we can keep it in unless it turns into a problem. In this diff it looks like we have new usages. Do you think we need these?